resize_data_part.target-sh

```diff
@@ -43,5 +43,16 @@ trap - EXIT
 mount_data_partition
 
 rm /media/data/local/etc/init.d/S00resize
+
+# Remove the /media/data/local directory if it's empty
+if rmdir /media/data/local/etc/init.d/; then
+	if rmdir /media/data/local/etc/; then
+		if rmdir /media/data/local/; then
+			# In an if statement to prevent set -e from stopping the script.
+			echo 'Removed /media/data/local'
+		fi
+	fi
+fi
```

Comment:
No need for this. The /media/data/local folder will be re-created just a moment afterwards...

Reply:
Good to know! Will remove before sending this for review (it's a chain of 3 PRs, still waiting for the other 2 to be merged)

Reply:
Done

Reply:
If we don't the `local` folder, it ends up being owned by the `default` user instead of `root`. It probably doesn't matter much.